src/main/kotlin/bakalarka/elearningplatform/model/ROLES.kt

```diff
@@ -0,0 +1,7 @@
+package bakalarka.elearningplatform.model
+
+enum class ROLES(val code: String) {
```

Comment:
IMHO by sa to malo volat `Roles`. https://kotlinlang.org/docs/enum-classes.html

Pri dalsom napisem iba NON_CAPS.